hudi/assets/service_checks.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "agent_version": "7.32.0",
+        "integration": "Hudi",
+        "groups": [
+            "host",
+            "instance"
+        ],
+        "check": "hudi.can_connect",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "name": "Can Connect",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored Hudi instance. Returns `OK` otherwise."
```

Comment:
I think it would be helpful to split up being able to connect and collect into two service checks. There was a support ticket for `oracle` where the customer wanted separate service checks for can connect and can collect/query because it was confusing to see `CRITICAL` but still being able to connect to the database.

Reply:
This would be an enhancement for JMX fetch itself, since that is where the service check is defined/emitted.

Reply:
Oh @coignetp  beat me to it 😆

Reply:
I'm not sure we can make that distinction in jmx checks